### PR TITLE
Fixes to Urania, Castelao and Erasmus

### PR DESCRIPTION
**Pull request by cvazquezs**
- Update to Erasmus and Urania entries.
- Addition of Castelao entry.

### CODE REVIEW

File: glossary/C.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The entire calorimetry system at LHCb («SPD»/«PS», «ECAL», «HCAL» is oft
 Its purpose is to discriminate between electrons, photons and hadrons as well as provide measurements of their energy and position.
 In Runs 1 and 2, the CALO played an important role providing transverse energy measurements to the «L0» trigger.
 
+## Castelao
+
+High-level physics analysis software application [repository](https://gitlab.cern.ch/lhcb/Castelao), similar to «Urania», but based on «DaVinci». This project hosts, among others, important packages used for the centralised production of «PID» and track calibration datasets. Any other packages which are based on «DaVinci» and need to be centrally distributed could be included in the project. Refer to the [repository](https://gitlab.cern.ch/lhcb/Castelao) for a list of available packages.
```
Comment (eduardo-rodrigues) [inline glossary/C.md]:
```suggestion
High-level physics analysis software repository [repository](https://gitlab.cern.ch/lhcb/Castelao), similar to «Urania», but based on «DaVinci». This project hosts, among others, important packages used for the centralised production of «PID» and track calibration datasets. Any other packages which are based on «DaVinci» and need to be centrally distributed could be included in the project. Refer to the [repository](https://gitlab.cern.ch/lhcb/Castelao) for a list of available packages.
```
It's not an executable, hence an application.

File: glossary/E.md [hunk 2]
```diff
@@ -37,13 +37,7 @@ Please see the entry for «LoI».
...
 ## Erasmus
 
-High-level physics analysis software package [repository](https://gitlab.cern.ch/lhcb/Erasmus),
-similar to «Urania», but based on «Bender»/«DaVinci».
-Refer to the [Twiki](https://twiki.cern.ch/twiki/bin/view/LHCb/Erasmus)
-for a list of the included packages, e.g. packages used by several analyst groups.
-
-Note that this project is now stopped, and retired from «CVMFS».
-All non-obsolete packages have been moved to the «Urania» project.
+High-level physics analysis software application [repository](https://gitlab.cern.ch/lhcb/Erasmus), similar to «Urania», but based on «Bender». Note that this project is now stopped, and retired from «CVMFS». All non-obsolete packages have been moved to the «Castelao» project, which is based instead on «DaVinci».
```
Comment (eduardo-rodrigues) [inline glossary/E.md]: Ditto.

File: glossary/C.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply glossary/C.md]: Else you can write "project".

File: glossary/U.md [hunk 3]
```diff
@@ -7,12 +7,7 @@ UFOs are known to cause LHC beam dumps upon detection, as a protective measure.
 
 ## Urania
 
-High-level physics analysis software package [repository](https://gitlab.cern.ch/lhcb/Urania),
-similar to «Erasmus:ERASMUS» (now retired), but without «Bender»/«DaVinci» as dependencies.
-This project is where crucial and central packages such as those for «PID»
-and track calibrations are to be found.
-Refer to the [repository](https://gitlab.cern.ch/lhcb/Urania)
-for a list of available packages.
+High-level physics analysis software application [repository](https://gitlab.cern.ch/lhcb/Urania), similar to «Castelao», but with the same low-level dependencies as the «Gaudi» project. This project is where crucial and central packages such as those used for «PID» response and track efficiency corrections are to be found. Refer to the [repository](https://gitlab.cern.ch/lhcb/Urania) for a list of available packages.
```
Comment (eduardo-rodrigues) [inline glossary/U.md]: Same comments as above.

Here PID and tracking related packages are mentioned just as for Castelao. Bette be super clear on the distinctions.

**Comment by eduardo-rodrigues** [comment]
Thanks a lot, @cvazquezs, for your updates following our chat. I've made some tin suggestions.

File: glossary/U.md (hunk 3 above)
Comment (cvazquezs) [reply glossary/U.md]: > Here PID and tracking related packages are mentioned just as for Castelao. Bette be super clear on the distinctions.

But the descriptions are different, no?

Comment (eduardo-rodrigues) [reply glossary/U.md]: I myself had to read both sentences again to notice the subtle differences. That's not that great, is it? I would suggest you give one or two example package names in each, to help.

Comment (cvazquezs) [reply glossary/U.md]: done

**Comment by eduardo-rodrigues** [comment]
Thanks a lot for the updates, @cvazquezs!

**Pull request merged**